CSS/Style.css

```diff
@@ -71,13 +72,19 @@ body{
 .search__section{
 
  padding: 10px;
- position:relative;  
+ position:absolute;  
  margin-left: 75%;
 }
 
 .robert{

```

Comment:
class .robert does not represent an actual element/functionality. I would suggest to use class/id naming for element purposes.

Please check out slide 110 in our presentation.